kitsune/products/views.py

```diff
@@ -43,12 +44,17 @@ def product_landing(request, template, slug):
         else:
             latest_version = 0
 
+    user_agent = request.META.get('HTTP_USER_AGENT', '')
+    browser = get_browser(user_agent)
+    show_fx_download = (product.slug == 'thunderbird' and browser != 'Firefox')
```

Comment:
You don't need to make any changes, but you should know the parenthesis here don't do anything. The just help readability. Which is a good thing, but it is important to know what is happening.